blocks/nav/nav.css

```diff
@@ -0,0 +1,9 @@
+.nav {
+  max-width: 100%;
+  background-color: #000D26;
+  position: fixed; 
```

Comment:
Are you sure that in desktop dimension it is fixed positioned? Maybe it shouldn't be in a such big screen?